packages/openapi-code-generator/src/typescript/common/schema-builders/abstract-schema-builder.ts

```diff
@@ -50,6 +53,7 @@ export abstract class AbstractSchemaBuilder<
     this.graph =
       parent?.graph ?? buildDependencyGraph(this.input, getSchemaNameFromRef)
     this.importHelpers(this.schemaBuilderImports)
+    this.typeBuilder = typeBuilder.withImports(this.schemaBuilderImports)
```

Comment:
Allows us to grab the type for a `Reference` without worrying about managing the imports